DigitalLearningSolutions.Web/Views/LearningMenu/Section/_TutorialCard.cshtml

```diff
@@ -3,34 +3,33 @@
 <div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
   <div class="nhsuk-card__content">
     <div class="nhsuk-grid-row">
-      <div class="@(Model.ShowRecommendationStatus ? "nhsuk-grid-column-two-thirds" : "nhsuk-grid-column-full")">
+      <div class="@(Model.ShowRecommendationStatus || Model.ShowLearnStatus ? "nhsuk-grid-column-two-thirds" : "nhsuk-grid-column-full")">
         <h3 class="nhsuk-card__heading card-with-status-heading" id="@Model.Id-name">
           @Model.TutorialName
         </h3>
       </div>
-      <div class="nhsuk-grid-column-one-third">
-        @if (Model.ShowRecommendationStatus)
-        {
+      @if (Model.ShowRecommendationStatus) {
+        <div class="nhsuk-grid-column-one-third">
           <strong class="status-tag @Model.StatusTagColour float-right-additional-information">
+            <span class="nhsuk-u-visually-hidden">This tutorial is</span>
             @Model.RecommendationStatus
           </strong>
-        }
-      </div>
-    </div>
-    <div class="nhsuk-grid-row">
-      <div class="@(Model.ShowLearnStatus ? "nhsuk-grid-column-two-thirds" : "nhsuk-grid-column-full")">
-        <div class="nhsuk-u-margin-bottom-4">
-          <partial name="Shared/_TutorialTimeSummary" model="@Model.TimeSummary" />
         </div>
-      </div>
-      <div class="nhsuk-grid-column-one-third">
-        @if (Model.ShowLearnStatus)
-        {
+      }
+      @if (Model.TimeSummary.ShowTime || Model.ShowRecommendationStatus) {
+        <div class="@(Model.ShowLearnStatus ? "nhsuk-grid-column-two-thirds" : "nhsuk-grid-column-full")">
+          <div class="nhsuk-u-margin-bottom-4">
+            <partial name="Shared/_TutorialTimeSummary" model="@Model.TimeSummary"/>
+          </div>
+        </div>
+      }
+      @if (Model.ShowLearnStatus) {
+        <div class="nhsuk-grid-column-one-third">
           <h3 class="nhsuk-heading-xs nhsuk-u-secondary-text-color float-right-additional-information">
             @Model.CompletionStatus
           </h3>
-        }
-      </div>
+        </div>
+      }
```

Comment:
Sorry, I'm confused. If both ShowRecommendationStatus and ShowLearnStatus are true, won't all these column sizes add up to more than 1? 🙂

Reply:
From the way the grid system works, It's fine to have columns add up to more than 1 as it will just start a new row as soon as one row is full. In the case where everything is true, there will be two-thirds for the title, one third for the recommendation status, two-thirds for the time and one third for the learn status. The time and the learn status will automatically drop onto a new row.
By not explicitly defining a new row for the time information and the learn status, it means that in the case were the recommendation status and time information are not displayed, the learn status will appear on the same row as the title and there won't be the white space that was found when testing